### PR TITLE
Add Online Academy solution page

### PR DESCRIPTION
**Pull request by michellusthof**
- [x] Add page description
- [x] Change client selection
- [x] SEO optimise text (in progress)
- [x] Add menu item + icon

### CODE REVIEW

**Comment by michellusthof** [comment]
[Preview](https://deploy-preview-2135--defacto-nl.netlify.com/online-academie)

File: source/images/icons/change-management.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...1v2H3v3H1v2H3v3H1v2H3v2H5V44.59A19.69,19.69,0,0,1,6.12,38H16v3H6v2H16v3H6v2H16v2h2c0-.69.05-1.35.09-2H20V46H18.31A23.06,23.06,0,0,1,19,43h1V41h-.34a27.63,27.63,0,0,1,5.06-8H28V31H26.56c1-1,2.08-2,3.16-3H34V26H31.91l.25-.22c1-.91,2-1.84,3-2.78H39V21H37.28c1-1,1.9-2,2.78-3H43V16H41.66a30.07,30.07,0,0,0,1.93-3H46V11H44.62a23.91,23.91,0,0,0,1.16-3H48V6H46.34a25.41,25.41,0,0,0,.47-3H49V1H47c0-.34,0-.66,0-1H45c0,.34,0,.67,0,1H35V3h9.81a23.17,23.17,0,0,1-.5,3H34V8h9.72a23.62,23.62,0,0,1-1.25,3H31v2H41.31a28.75,28.75,0,0,1-2.15,3H27v2H37.44c-1,1-2,2-3,3H21v2H32.25c-.47.43-.93.86-1.41,1.28S29.59,25.43,29,26H19v2h7.81c-1,1-2,2-2.93,3H19v2h3.19A30.69,30.69,0,0,0,20,36H18V22.72l1.16-1.06c3.61-3.27,7.29-6.44,10.09-9.91S34,4.44,34,0H32c0,.34,0,.67,0,1H30V3h1.62a14.39,14.39,0,0,1-1,3H28V8h1.5a27.15,27.15,0,0,1-1.81,2.5c-.14.17-.3.33-.44.5H25v2h.5c-1,1-2.05,2-3.12,3H18V13h2V11H18V8h2V6H18V3h2V1H18V0H16V1H6V3H16V6H6V8H16v3H6v2H16v3H6v2H16v3H6v2h8.78c-1,1-2,2-3,3H6v2H9.94A31.86,31.86,0,0,0,5,35.47V0ZM18,18h2.22L18,20Zm-2,6.59V26H14.56ZM12.56,28H16v3H10.06C10.84,30,11.67,29,12.56,28Zm-3.9,5H16v3H7A24,24,0,0,1,8.66,33ZM18,38h.84c-.3.59-.58,1.2-.84,1.81Z" transform="translate(-1 0)" fill="#fff"/></svg>
```
Comment (RonaldDijkstra) [inline source/images/icons/change-management.svg]: width and height should not be in mm

**Comment by RonaldDijkstra** [comment]
What kind of icon do you want for this?

**Comment by michellusthof** [comment]
> What kind of icon do you want for this?

Something like this?

File: source/localizable/online-academie/_clients.nl.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <%= image_tag "clients/colour/ambulancezorg-groningen.png", alt: "Ambulancezorg Groningen", title: "Ambulancezorg Groningen" %>
+  </div>
+  <div class="client">
+    <%= image_tag "clients/colour/logo-zinn.svg", alt: "ZINN", title: "ZINN", class: "client-logo" %>
+  </div>
+  <div class="client">
+    <%= image_tag "clients/colour/logo-sheerenloo.svg", alt: "'s Heeren Loo", title: "'s Heeren Loo", class: "client-logo" %>
+  </div>
+  <div class="client">
+    <%= image_tag "clients/colour/logo-delelie.svg", alt: "De Lelie Zorggroep", title: "De Lelie Zorggroep", class: "client-logo" %>
```
Comment (RonaldDijkstra) [inline source/localizable/online-academie/_clients.nl.html.erb]: Not here but would make more sense to rename this to logo-de-lelie?

Comment (michellusthof) [reply source/localizable/online-academie/_clients.nl.html.erb]: Yeah, it does

**Pull request merged**